packages/amplify-category-auth/src/provider-utils/awscloudformation/handlers/resource-handlers.ts

```diff
@@ -135,7 +135,7 @@ export const getUpdateAuthHandler = (context: $TSContext) => async (request: Ser
   const envSpecificParams: $TSAny = {};
   const cliInputs = { ...sharedParams };
   ENV_SPECIFIC_PARAMS.forEach(paramName => {
-    if (paramName in request) {
+    if (paramName in cliInputs) {
```

Comment:
Are `request` keys a subset of `cliInputs`?  How do we know that we're not going to be missing stuff from `request` now?

Would it be safer to use something like:
```if (paramName in request || paramName in cliInputs) {```

Reply:
Yes `request` is a subset of `cliInputs` which was the cause of the error. There was a key that should have been removed in the `cliInputs` object but it was not present in the `request` object, thus it made it through this filter.

Even if it was not a subset, line 140 would do nothing if the key was in `request` but not in `cliInputs`